### DISCUSSION TITLE
An error occurred while running

### DISCUSSION THREAD

**Discussion by Astronya** [icosa-gallery/open-brush#608] [Q&A]
Why did I get an error when running OpenBrush in Unity? There is no problem with compilation, but there is an error in the diagram when hitting play run. I only downloaded main, not other related packages.I just imported main's package into Unity3d version 2021.3.33. And TMP was introduced.
![errorpicture](https://github.com/icosa-foundation/open-brush/assets/99575490/680e8180-c15d-4286-b99e-e8af0b730c5e)

**Comment by andybak** [comment]
>  I only downloaded main, not other related packages.

> I just imported main's package into Unity3d version 2021.3.33. 

Can you clarify what these two steps? What did you import exactly?

This repo is an entire Unity project. It should be opened directly as a Unity project. Am I misunderstanding what you did?

Also - what OS/hardware are you running on?

Also - please use  Unity3d version 2021.3.30 - it's probably not going to make a big difference but it's always best to stick to the exact Unity version for a complex project like this.

**Reply by Astronya** [reply]
Hello, I only downloaded and used open-rush-main. I use the open function of UnityHub to directly open this project. There were no errors during compilation, but an error occurred during runtime.

I run it on the PC side of Windows 10.

**Reply by andybak** [reply]
I don't get any errors. The only difference I think is that I'm using 2021.3.30

What happens if you just ignore the errors and continue as normal?
